### ISSUE TITLE
Specialty Staff Line

### ISSUE THREAD

**Issue by Fourniture** [musescore/MuseScore#25460]
### Your idea

Sometimes composers want to specify pitchless sounds on a pitched instrument (e.g. taps on a cello's body). The customary way of notating this is to have a percussion-type line appear above:
![image](https://github.com/user-attachments/assets/c6ab90e5-f0c9-4240-94b9-8a7da1a037f5)
This should be input in the same manner as described in post #24545, except there would now be another option in the right-click menu, titled, "Add pitchless line", or something similar.

### Problem to be solved

Musescore cannot notate this.

### Additional context

Please also see post #24629.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by cbjeukendrup** [comment]
I think this can be achieved using a staff type change, from the Layout palette.

**Comment by Dima-S-Jr** [comment]
> I think this can be achieved using a staff type change, from the Layout palette.

Obviously, a "Specialty Staff Line" means a line separate from the main staff and having a different distance (greater) than between the lines of the main staff. This cannot be done with the help of "staff type change". It's more like a "linked staff" or something like that. 

In general, there really is a point in such a notation. Thus, it would be nice to experiment with various techniques on woodwinds, brass, keyboards, etc.

**Comment by cbjeukendrup** [comment]
That was absolutely not obvious from the example image, but ok.

Creating extra staves is already possible using the instruments panel. The number of lines of the extra staff can be set to 1 in the staff/part properties dialog. It looks like the only thing that's necessary would be a special form of "hide empty staves" that does not show/hide the staff per _system_ but per _measure_.

**Comment by Fourniture** [comment]
The point is that I requested a dedicated feature that would do this, not a workaround. #24545 also lays out a process for adding any elements to a staff, so it can be expanded in the future to include more things besides ossia and a pitchless line. Remember, ossia are not, strictly speaking, an extra stave; they are a snapshot of another option for the performer. A pitchless line is also not an extra staff, but rather a staff extension for a small period. There are times when whole measures (of a violin or trumpet part, etc.) are converted to a pitchless percussion stave for an extended period, but this idea is for times when all that is needed is a staff extension.

**Comment by MarcSabatella** [comment]
I think you may be confusing three different notations here:

- non-pitched notes to be played directly are normally notated directly on the staff, no additional line required. This is already supported by merely changing noteheads and/or using instrument changes.
- non-pitched notes meant to indicate an accompaniment rhythm in addition to the normal notes are normally notated above the staff *without* any additional staff line.  This is supported as “rhythmic slash notation”.
- notation of an alternate version of measure (whether percussive of not, whether one live of five or seventeen - is called an “ossia”).  This is already supposed via the “cutaway” option for the staff properties, although no doubt, a more direct ossia feature would be nice as well.

It isn’t really clear what you are asking for above and beyond these things that are already supported.  Do you have a published example to demonstrate?

**Comment by Fourniture** [comment]
From Music Notation in the Twentieth Century, by Kurt Stone:
![20241108_111905](https://github.com/user-attachments/assets/0967d155-90a4-470d-a24d-adc908cc3279)

**Comment by MarcSabatella** [comment]
So, how does this differ from an ossia that just happens to show a percussion staff?  What *additional* feature are you requesting that is *not* already possible by simply adding a cutaway percussion staff or that would not already be be solved even better by a dedicated ossia feature?

**Comment by Fourniture** [comment]
You are asking me why I want a feature for pitchless percussion lines, when we could (1) add an extra staff, cut away measures, and still have to deal with formatting issues; or (2) use an ossia feature for them, when that is _not_ what the ossia is for. 
   I ask for a dedicated feature that does what it is supposed to do, is only used for what it is meant for, and can be expanded to include other options. It doesn't matter whether ossia and pitchless lines use the same code; it matters whether the user sees options for what he wants to do. This is user experience 101. If Musescore continues to want to use workarounds, they will never compare to Dorico or Sibelius.

**Comment by szeitlin** [comment]
FWIW I could use a feature like this for jazz lead sheets, too, and have been struggling to find a workaround that isn't going to confuse the band.

**Comment by MarcSabatella** [comment]
This is what rhythmic slash notation is for - use voice 3 and make sure to only select that voice when running the command.  If you need further assitance learning how to do this, please ask for help ion the Support forum at musescore.org.
